modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,296 @@
+
+
+[IMPORTANT]
+====
+The same version of the applications must function on both the current and the target release of {product-title}.
+====
+--
+
+. Generate the seed image by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f seedgenerator.yaml
+----
+
++
+[IMPORTANT]
+====
+The cluster loses API capabilities while the {lcao} generates the seed image.
+====
+
++
+Applying the `SeedGenerator` CR stops the `kubelet` and the CRI-O operations, and then it starts the image generation.
+Finally, the {lcao} starts a cleanup process that allows you to keep the seed {sno} fully functional.
+
+.Verification
+
+. You can monitor the image generation process from the target cluster.
+
+.. SSH into the target cluster:
++
+[source,terminal]
+----
+$ ssh <node_ID>
+----
+// must have an ssh key configured via MC. When do we do this and how?
+
+.. List the running containers:
++
+--
+[source,terminal]
+----
+$ podman ps
+----
+
+.Example output
+[source,terminal]
+----
+CONTAINER ID  IMAGE                                             COMMAND               CREATED         STATUS         PORTS       NAMES
+f9484b2e64dc  quay.io/<user>/lifecycle-agent-operator:latest  create --authfile...  31 seconds ago  Up 31 seconds              ibu_imager
+----
+--
+
+.. Check the image generation logs by running the following command:
++
+[source,terminal]
+----
+$ podman logs -f ibu_imager
+----
+
+. Check the status of the `SeedGenerator` CR:
++
+--
+[source,terminal]
+----
+$ oc get seedgenerator -A -oyaml
+----
+
+.Example output
+[source,yaml]
+----
+status:
+    conditions:
+    - lastTransitionTime: "2024-01-01T09:00:00Z"
+      message: Seed Generation completed
+      observedGeneration: 1
+      reason: Completed
+      status: "True"
+      type: SeedGenCompleted <1>
+    - lastTransitionTime: "2024-01-01T09:02:00Z"
+      message: Seed Generation completed
+      observedGeneration: 1
+      reason: Completed
+      status: "False"
+      type: SeedGenInProgress
+    observedGeneration: 1
+----
+<1> The seed image generation is complete.
+--
+
+. Verify that the {sno} is running and is attached to the {rh-rhacm} hub cluster:
++
+--
+[source,terminal]
+----
+$ oc get nodes,clusterversion,co
+----
+
+.Example output
+[source,terminal]
+----
+NAME                                            STATUS   ROLES                         AGE   VERSION
+node/snonode.sno-worker-02.example.redhat.com   Ready    control-plane,master,worker   12m   v1.27.6+d548052
+
+NAME                                         VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+clusterversion.config.openshift.io/version   4.14.4    True        False         11h     Cluster version is 4.14.4
+----
+
+[NOTE]
+====
+The {sno} node requires time to recover after restarting the `kubelet` operation.
+====
+--
+
+[id="ztp-image-based-upgrade-additional-seed-image-generation_{context}"]
```

Comment:
Is this true? cc/ @browsell  @omertuc  - I understood that it is not going to be supported.

Reply:
True, section can be removed. Seed clusters should be considered spoiled after they live for more than a day or so. Right now it's in theory, in the future it'll be a hard error once we make a final decision about it.